.env.template

```
@@ -3,11 +3,8 @@ FLASK_APP=todo_app/app
 FLASK_ENV=development
 
 # Change the following values for local development.
-token=token-key
-api_key=api-key
-
-done_status=done-status
-doing_status=doing-status
-toDo_status=toDo-status
-
-TRELLO_BOARD_ID=board-id
\ No newline at end of file
+MONGO_URL=URL
+MONGO_USER=USERNAME
+MONGO_PASSWORD=PASSWORD
+MONGO_DB=DATABASE
```

Think you're missing one variable
```suggestion
MONGO_PASSWORD=PASSWORD
MONGO_DB=DATABASE
MONGO_PROTOCOL=mongodb://
```